src/python/pants/backend/python/docs/sphinx/sphinx_subsystem.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from pants.backend.python.goals import lockfile
+from pants.backend.python.goals.export import ExportPythonTool, ExportPythonToolSentinel
+from pants.backend.python.goals.lockfile import GeneratePythonLockfile
+from pants.backend.python.subsystems.python_tool_base import ExportToolOption, PythonToolBase
+from pants.backend.python.subsystems.setup import PythonSetup
+from pants.backend.python.target_types import ConsoleScript
+from pants.core.goals.generate_lockfiles import GenerateToolLockfileSentinel
+from pants.engine.rules import collect_rules, rule
+from pants.engine.unions import UnionRule
+from pants.util.docutil import git_url
+
+
+class SphinxSubsystem(PythonToolBase):
+    options_scope = "sphinx"
+    name = "sphinx"
+    help = "The Python documentation generator (https://www.sphinx-doc.org/en/master/index.html)."
+
+    default_version = "sphinx>=4.5.0,<4.6"
```

Comment:
Any reason not to go with `<5` ? This seems a little restrictive

Reply:
The lockfile pins a particular version anyway: better not to claim that we "might" be using something that we aren't.